src/test/java/vention/pagesTest/LoginFlowTest.java

```diff
@@ -0,0 +1,50 @@
+package vention.pagesTest;
+
+import vention.driver.DriverManager;
+import vention.entity.User;
+import vention.entity.UserFactory;
+import vention.pages.CookiePopup;
+import vention.pages.PreLoginPage;
+import vention.pages.LoginPage;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import org.testng.Assert;
+
+
+public class LoginFlowTest extends BaseTest {
+  private CookiePopup cookiePopup;
+  private PreLoginPage preLoginPage;
+  private LoginPage loginPage;
+
+  @BeforeMethod
+  public void beforeMethod() {
+    preLoginPage = new PreLoginPage();
+    preLoginPage.openPage();
+
+    cookiePopup = new CookiePopup();
+    cookiePopup.clickRejectAllButton();
+
+    preLoginPage.clickLoginButton();
+
+    loginPage = new LoginPage();
+  }
+
+  @Test
+  public void testCompleteUserLoginFlow() {
+    SoftAssert softAssert = new SoftAssert();
+    softAssert.assertTrue(loginPage.isEmailInputDisplayed(), "Email input should be visible");
+    softAssert.assertTrue(loginPage.isPasswordInputDisplayed(), "Password input should be visible");
+    softAssert.assertTrue(loginPage.isNextButtonDisplayed(), "Next button should be visible");
+    softAssert.assertAll();
+ 
+    User existingUser = UserFactory.getExistingUser();
+    loginPage.enterLoginCredentials(existingUser.getEmail(), existingUser.getPassword());
```

Comment:
I'd recommend to pass User object to the method. It increases maintainability of the method

If in the future you will need to pass new parameters to the method, you will have to add new arguments to the method (with current approach) but if you pass User object as an argument, in case of any new user arguments you will only need to process them inside the method. Method signature won't change